site/content/docs/5.0/examples/offcanvas-navbar/index.html

```diff
@@ -12,7 +12,7 @@
 <nav class="navbar navbar-expand-lg fixed-top navbar-dark bg-dark" aria-label="Main navigation">
   <div class="container-fluid">
     <a class="navbar-brand" href="#">Offcanvas navbar</a>
-    <button class="navbar-toggler p-0 border-0" type="button" data-bs-toggle="offcanvas" aria-label="Toggle navigation">
+    <button class="navbar-toggler p-0 border-0" type="button" data-bs-toggle="custom-offcanvas" aria-label="Toggle navigation">
```

Comment:
If `data-bs-toggle` is just to select the button, you should prefer any other selector since `data-bs-toggle` is generally used to target some triggers and it may give a wrong understanding to users IMO (since `data-bs-toggle="custom-offcanvas"` is not a selector in bootstrap).

Reply:
Just rename it to something that's it's clearer that affects only the button :)